functionary/ui/templates/partials/token.html

```diff
@@ -0,0 +1,18 @@
+<div id="token">
+    <strong class="fs-4"><i class="fa fa-id-badge fa-sm fa-fw pe-1"></i>Token</strong>
+    <div class="col-9">
+        <form >
+            {% csrf_token %}
+            <span class="col-7" data-token="{{ token.key }}">{{ token.key }}</span>
```

Comment:
It might be good to just show •••••••••••••• or something similar and then have a button for revealing the token, rather than just showing it by default.  It is a secret after all.

Reply:
Added a button to swap between masked and unmasked.